resources/mmex.desktop

```diff
@@ -1,33 +1,37 @@
+Name[ru]=MMEX Mенеджер личных финансов
 Exec=mmex
 Type=Application
 Keywords=money;accounts;accounting;finance;
-Name[ru]=MMEX Mенеджер личных финансов
-GenericName=Cross Platform Personal Financial Manager
+Keywords[hu]=pénz;számlák;számvitel;pénzügyek;
+GenericName=Personal financial and money management software
+GenericName[hu]=Személyes pénzügyi és pénzkezelési szoftver
 GenericName[it]=Gestore multi-piattaforma di finanza personale
 GenericName[ru]=Кросплатформенный менеджер личных финансов
-Comment=A free, open-source, cross-platform, easy-to-use personal finance software
-Comment[it]=Un gestore delle finanze personali, libero, open source, facile da usare, multi piattaforma.
-Comment[ru]=Бесплатная и лёгкая в использовании, мультиплатформенная программа с открытым кодом.
+Comment=Organize finances and track cash flow
 Comment[cs]=Zdarma, open-source, cross-platformní, bezstarostný-až k-cvičení osobní finance software
 Comment[de]=Ein freies, Open-Source, Cross-Plattform, einfach zu bedienende persönliche Finanz-Software
 Comment[es]=Un libre, de código abierto, multiplataforma, fácil de usar software de finanzas personales
 Comment[fi]=Ilmainen, avoimen lähdekoodin, cross-platform, helposti käyttää henkilökohtaisia rahoittaa ohjelmisto
 Comment[fr]=Une libre, open-source, multi-plateforme, facile à utiliser un logiciel de finances personnelles
-Comment[hu]=Az ingyenes, nyílt forráskódú, platformfüggetlen, könnyen használható személyes pénzügy szoftver
+Comment[hu]=Pénzügyek megszervezése és pénzforgalom nyomon követése
 Comment[is]=A frjáls, opinn-uppspretta, kross-pallur, þægilegur-til-nota persónuleg fjármál hugbúnaður
+Comment[it]=Un gestore delle finanze personali, libero, open source, facile da usare, multi piattaforma.
 Comment[lv]=Bezmaksas, atvērtā koda, cross-platformu, viegli lietojamu personīgās finanšu programmatūru
 Comment[nl]=Een vrije, open-source, cross-platform, makkelijk te gebruiken persoonlijke financiële software
 Comment[no_NY]=A free, open-source, cross-platform, easy-to-use personal finance software
 Comment[pt]=Um livre, open-source e multi-plataforma, fácil de usar software de finanças pessoais
 Comment[pt_BR]=Um livre, open-source e multi-plataforma, fácil de usar software de finanças pessoais
 Comment[ro]=Un gratuit, open-source, cross-platform, usor de folosit software-ul finanţelor personale
+Comment[ru]=Бесплатная и лёгкая в использовании, мультиплатформенная программа с открытым кодом.
 Comment[sk]=Slobodný, open-source, viacplatformový, jednoducho použiteľný softvér na správu osobných financií
-Comment[sl]=A free, open-source, cross-platform, easy-to-use personal finance software
+Comment[sl]=Organizirajte finance in spremljajte denarni tok
 Comment[tr]=Ücretsiz, açık kaynak, çapraz platform, kolay kişisel finans yazılımı kullanma
-Comment[uk]=A free, open-source, cross-platform, easy-to-use personal finance software
+Comment[uk]=Організуйте фінанси та відстежуйте грошові потоки
```

Comment:
This doesn't look like the correct string for UK

Reply:
Sorry, thought it was Ukraine, not United Kingdom. Should be fixed now.

Reply:
Should be fixed now as `z` has been changed to `s`

Reply:
Is it meant to be `Ukraine`?
https://developers.google.com/admin-sdk/directory/v1/languages

en-GB : English (UK)
uk : Ukrainian

Reply:
MMEX file uses an underscore `_`, yet https://developers.google.com/admin-sdk/directory/v1/languages uses a hyphen `-`. Which one is correct?

Reply:
I think you are correct that uk is Ukrainian, my apologies. I wasn't thinking in terms of ISO 639 codes and the prior string being in English threw me off.

Reply:
Seems like underscore `_` is correct based on https://docs.flatpak.org/en/latest/flatpak-command-reference.html

> I think you are correct that uk is Ukrainian, my apologies. I wasn't thinking in terms of ISO 639 codes and the prior string being in English threw me off.

No worries, best to communicate until we are all on the same page.

Should the translations be updated using computer translation?

Reply:
Thunderbird uses en-GB instead of en-UK.
https://www.thunderbird.net/en-GB/

Perhaps en_UK is incorrect and should be en_GB?

Reply:
Seems that `en_GB` is the correct locale for British English.

```
$ locale -a
C
C.utf8
en_AG
en_AU.utf8
en_GB.utf8
en_IL
en_IN
en_NG
en_US.utf8
en_ZM
hu_HU.utf8
POSIX
```